### ISSUE TITLE
Create trash meter

### ISSUE THREAD

**Issue by AutoParanoid** [AutoParanoid/trash-island#6]

**Comment by manno-xx** [comment]
I think the UI Image element can be somewhat easier. Using the slider would require you to modify it (remove the 'needle). If you set the Image element to use another sprite than the default, you can set it's type to fill which allows you to alter its width (internally a mask I think)

**Comment by eloyrobillard** [comment]
Right! I had totally forgotten about this method, but it's one of these Unity manuals I bought a while ago.  \
Thank for the +XP.
